kite-data/kite-data-core/src/main/java/org/kitesdk/data/Formats.java

```diff
@@ -53,10 +53,18 @@ private Formats() {
       new CompressionType[] { Uncompressed });
 
   /**
-   * Return a {@link Format} for the format name specified. If
-   * {@code formatName} is not a valid name, an IllegalArgumentException is
-   * thrown. Currently the formats <q>avro</q>, <q>csv</q>, and <q>parquet</q>
-   * are supported. Format names are case sensitive.
+   * INPUTFORMAT: a mapreduce InputFormat (read-only).
```

Comment:
I've added validation checks in partitioned writer and file writer to ensure that a bad format fails in initialize.